aptos-move/aptos-release-builder/src/components/feature_flags.rs

```diff
@@ -131,6 +131,7 @@ pub enum FeatureFlag {
     FederatedKeyless,
     TransactionSimulationEnhancement,
     CollectionOwner,
+    TransactionContextHashFunctionUpdate,
```

Comment:
Reading through Igor's comments, I see Igor has suggested some approach without a feature flag. Are we still using this feature flag?